src/main/java/bank/currencyconverter/CurrencyController.java

```diff
@@ -0,0 +1,21 @@
+package bank.currencyconverter;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Currency;
+
+@RestController
+@RequestMapping("/currency")
+@RequiredArgsConstructor
+public class CurrencyController {
+    private final CurrencyConverter currencyConverter;
+
+    @GetMapping("/convert/{from}/{to}/{amount}")
```

Comment:
```suggestion
    @GetMapping("/convert")
```
воно не прийнято так в ресті 
тож краще або через queryParams або в body (для пост запиту)